frontend/javascript/components/moderation/moderation-dashboard.vue

```diff
@@ -180,13 +181,16 @@ export default {
     this.room = new Room(this.config.url.moderationWebsocket)
     getData(this.config.url.listReports).then((data) => {
       this.reports = [...this.reports, ...data.objects]
+      this.reportCount = data.meta.total_count
     })
     this.room
       .connect()
       .on('userlist', (data) => {
         this.moderators = data.userlist
       })
       .on('report_added', (data) => {
+        this.reportCount +=
```

Comment:
I assume this check is in place, so that the report count doesn't change if the report is already present and accounted for? Maybe this makes it more clear:


```suggestion
        if (this.reports.some(r => r.id === data.report.id)) {
        	this.reportCount += 1
        }
```

Likewise below.